### ISSUE TITLE
[Bug] MYRIAD unable to run

### ISSUE THREAD

**Issue by Averen19** [openvinotoolkit/openvino#11476]
Hello, I'm trying to run a yolov5 model with openvino and NCS2. My assumption with installing openvino is that it comes preinstalled with NCS2 dependencies, but when I'm trying to run with NCS2 I get the following issue:

![image](https://user-images.githubusercontent.com/93975422/161906376-94f7d285-8ed1-4f24-b4df-6501c86bd431.png)

**Comment by mateusztabaka** [comment]
Hi, have you tried to run setupvars.bat script before running the inference?

**Comment by Averen19** [comment]
@mateusztabaka  yes I have. I am able to run with my cpu but not with MYRIAD. It seems that the openvino 2022 does not have the inference engine folder which was found in the 2021 version, so I do not know where to install the myriad driver neccessary.

**Comment by Averen19** [comment]
Hi, whats the difference between your link and the one available from intel directly [https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/download.html] I tried to download your version but it says I already have the files.

**Comment by dkurt** [comment]
Oh, that's the same but just a direct link. I just checked my Windows install can find myriad plugin:
```
cache.json                       openvino_intel_gpu_plugin.dll*
gna.dll*                         openvino_intel_hddl_plugin.dll*
openvino.dll*                    openvino_intel_myriad_plugin.dll*
openvino_auto_batch_plugin.dll*  openvino_ir_frontend.dll*
openvino_auto_plugin.dll*        openvino_onnx_frontend.dll*
openvino_c.dll*                  openvino_paddle_frontend.dll*
openvino_gapi_preproc.dll*       openvino_tensorflow_fe.dll*
openvino_hetero_plugin.dll*      pcie-ma2x8x.elf
openvino_intel_cpu_plugin.dll*   plugins.xml
openvino_intel_gna_plugin.dll*   usb-ma2x8x.mvcmd
```

Can you please share exact commands which are used? Try on a new prompt:
```
call "C:\Program Files (x86)\Intel\openvino_2022\setupvars.bat"
```
then verify that OpenVINO Python is from `C:\Program Files (x86)\Intel\openvino_2022`:
```python
import openvino
print(openvino__file__)
```
should be:
```
C:\Program Files (x86)\intel\openvino_2022\python\python3.8\openvino\__init__.py
```
(3.8 or other Python version)

**Comment by Averen19** [comment]
I cant seem to find the setupvars.bat found in previous versions of Openvino, and in the 2022 version it doesnt have folders for inference engine or model optimizer either.
![image](https://user-images.githubusercontent.com/93975422/162006053-3ac545e6-202a-4b52-83be-47d632ea2e28.png)

**Comment by dkurt** [comment]
Seems like it's not properly installed - try double-click on `w_openvino_toolkit_p_2022.1.0.643_offline.exe` and follow install GUI or by the following command:
```
start /WAIT w_openvino_toolkit_p_2022.1.0.643_offline.exe -s -a --silent --eula accept
```

**Comment by Averen19** [comment]
I have tried reinstalling again, and to ensure that I did install with Movidius, the following image can be referrenced.
![image](https://user-images.githubusercontent.com/93975422/162012417-078a5269-0420-4b3a-bcf6-80b6f5700ef5.png)

I tried your example and got the following:
![image](https://user-images.githubusercontent.com/93975422/162012483-7dde429d-92a2-40a7-9cca-87c6e470d297.png)

**Comment by jgespino** [comment]
@Averen19 I will work with you on your original issue here: https://github.com/openvinotoolkit/openvino/issues/11458 for the raspberry pi package. Closing this issue as your MYRIAD issue has been resolved for Windows.
